modules/core/shared/src/main/scala/eu/timepit/refined/api/Min.scala

```diff
@@ -0,0 +1,51 @@
+package eu.timepit.refined.api
+
+trait Min[FTP] { def min: FTP }
+object Min {
+  def apply[FTP](implicit ev: Min[FTP]): Min[FTP] = ev
+  def instance[FTP](m: FTP): Min[FTP] = new Min[FTP] { def min: FTP = m }
+}
+
+trait Max[FTP] { def max: FTP }
+object Max {
+  def apply[FTP](implicit ev: Max[FTP]): Max[FTP] = ev
+  def instance[FTP](m: FTP): Max[FTP] = new Max[FTP] { def max: FTP = m }
+}
+
+/**
+ * Auxiliary type class that provides the next smaller or next greater
+ * value for a given argument. This is needed for the `Arbitrary`
+ * instances of numeric refined types.
+ */
+trait Adjacent[T] {
```

Comment:
Could you move this into the `internal` package. I guess we don't want to advertise `Adjacent` as part of our API? Also the Scaladoc needs to be updated. It is not about supporting `Arbitrary` instances any more, right?